news/2022-07-25-mappers-guild-updates.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: post
+title: Changes within the Mappers' Guild
+date: 2022-07-25 10:00:00 +0000
+---
+
+The **Mappers' Guild** dishes out some exciting news for anyone interested in mapping contests, Featured Artists, beatmap pack medals, and a bunch more!
+
+![](/wiki/shared/news/banners/mappersguild.png)
+
+Never heard of the [**Mappers' Guild**](/wiki/Community/Mappers_Guild)?
+
+The guild is a way for mappers to earn rewards for mapping songs from osu!'s [Featured Artists](https://osu.ppy.sh/beatmaps/artists). In the last 5 years, the **Mappers' Guild** expanded to [its own website](https://mappersguild.com) and includes plenty of extra mapping-related opportunities!
+
+This article covers the biggest changes to the **Mappers' Guild** since [our last homepage update](https://osu.ppy.sh/home/news/2020-11-20-featured-artist-beatmap-updates-from-the-mappers-guild). The topics are unrelated to each other, so feel free to skip around:
```

Comment:
```suggestion
This article covers the biggest changes to the **Mappers' Guild** since [our last home page update](https://osu.ppy.sh/home/news/2020-11-20-featured-artist-beatmap-updates-from-the-mappers-guild). The topics are unrelated to each other, so feel free to skip around:
```